### ISSUE TITLE
Party Bugs

### ISSUE THREAD

**Issue by TheWylot** [andrei1058/BedWars1058#70]
**Describe the bug**
Party system problems with new groups

**To Reproduce**
Actually i created a new map group 1v1, when two players in a  party joins, arena is max but cooldown is not started and Scoreboard shows Waiting...

**System:**
 - OS: Windows Server 2019
 - Server software: PaperSpigot
 - Server Version : 1.8.8
 - Plugin Version: 21.11

**Comment by Abolfazl-Shaban** [comment]
hey this problem does not fixed!

**Comment by Abolfazl-Shaban** [comment]
@andrei1058 please fix that

**Comment by andrei1058** [comment]
@Abolfazl-Shaban is it still happening today? with the latest bedwars version?

**Comment by Abolfazl-Shaban** [comment]
> @Abolfazl-Shaban is it still happening today? with the latest bedwars version?

Yes, I tried with the latest version last month and it still had a problem, so far you haven't updated it and it hasn't been fixed

**Comment by andrei1058** [comment]
> > @Abolfazl-Shaban is it still happening today? with the latest bedwars version?
> 
> Yes, I tried with the latest version last month and it still had a problem, so far you haven't updated it and it hasn't been fixed

I did update it, latest version is not 21.11 as you mentioned.

Bedwars1058 v**23.12.2** [here](https://www.spigotmc.org/resources/bedwars1058-opensource.97320/).
